pkg/wakaama/include/lwm2m_platform.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2019 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     pkg_wakaama
+ * @defgroup    lwm2m_platform Platform adaption for Wakaama package
+ * @brief Adaption of Wakaama LwM2M package to RIOT
+ * @{
+ * @file
+ * @brief Definitions and public API for Wakaama adaption layer
+ *
+ * @author      Leandro Lanzieri <leandro.lanzieri@haw-hamburg.de>
+ */
+#ifndef LWM2M_PLATFORM_H
+#define LWM2M_PLATFORM_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @defgroup lwm2m_platform_conf Wakaama LwM2M platform adaption
+ * @ingroup config
+ * @brief Compile-time configuration options for the Wakaama LwM2M platform
+ *        adaption layer.
+ * @{
+ */
+/** @brief Size of allocation buffer in bytes */
+#ifndef LWM2M_TLSF_BUFFER
+#define LWM2M_TLSF_BUFFER     5120
```

Comment:
Much clearer to express this value in bytes here and translate internally!  With this value, I now see 416 bytes used, and 1452 available on a samr21-xpro after connecting to the server. Seems reasonable to me.